gulpfile.js

```diff
@@ -4,7 +4,8 @@
 const gulp = require('gulp'),
     del = require('del'),
     vinylPaths = require('vinyl-paths'),
-    $ = require('gulp-load-plugins')();
+    $ = require('gulp-load-plugins')(),
+    sourcemaps = require('gulp-sourcemaps'),
```

Comment:
Why not `$.sourcemaps`?

Reply:
I am totally new to Gulp although I like it very much. I'm a PHP developer so bare with me :)